sqlathanor/declarative/_base_configuration_mixin.py

```diff
@@ -583,49 +583,54 @@ def get_serialization_config(cls,
+
+            if config_set and config_set not in cls.__serialization__:
+                raise ValueError('%s config set (%s) not found in __serialization__ '
+                                 ' configuration' % (cls, config_set))
+
+            if config_set:
+                __serialization__ = [x for x in cls.__serialization__[config_set]]
+            else:
+                __serialization__ = [x for x in cls.__serialization__]
+
+            declarative_attributes = cls._get_declarative_serializable_attributes(
+                from_csv = from_csv,
+                to_csv = to_csv,
+                from_json = from_json,
+                to_json = to_json,
+                from_yaml = from_yaml,
+                to_yaml = to_yaml,
+                from_dict = from_dict,
+                to_dict = to_dict,
+                exclude_private = exclude_private
+            )
+            meta_attributes = cls._get_meta_serializable_attributes(
+                from_csv = from_csv,
+                to_csv = to_csv,
+                from_json = from_json,
+                to_json = to_json,
+                from_yaml = from_yaml,
+                to_yaml = to_yaml,
+                from_dict = from_dict,
+                to_dict = to_dict,
+                config_set = config_set
+            )
 
-        if config_set and config_set not in cls.__serialization__:
-            raise ValueError('config set (%s) not found in __serialization__ '
-                             ' configuration' % config_set)
+            attributes = [x for x in meta_attributes]
+            attributes.extend([x for x in declarative_attributes
+                               if x not in attributes and x not in __serialization__])
 
-        if config_set:
-            __serialization__ = [x for x in cls.__serialization__[config_set]]
-        else:
-            __serialization__ = [x for x in cls.__serialization__]
+            return attributes
 
-        declarative_attributes = cls._get_declarative_serializable_attributes(
-            from_csv = from_csv,
-            to_csv = to_csv,
-            from_json = from_json,
-            to_json = to_json,
-            from_yaml = from_yaml,
-            to_yaml = to_yaml,
-            from_dict = from_dict,
-            to_dict = to_dict,
-            exclude_private = exclude_private
-        )
-        meta_attributes = cls._get_meta_serializable_attributes(
-            from_csv = from_csv,
-            to_csv = to_csv,
-            from_json = from_json,
-            to_json = to_json,
-            from_yaml = from_yaml,
-            to_yaml = to_yaml,
-            from_dict = from_dict,
-            to_dict = to_dict,
-            config_set = config_set
-        )
+        k = f'attributes_fc{str(from_csv)}tc{str(to_csv)}fj{str(from_json)}tj{str(to_json)}fy{str(from_yaml)}ty{str(to_yaml)}fd{str(from_dict)}td{str(to_dict)}ep{str(exclude_private)}',
```

Comment:
Unfortunately, this syntax for literal string interpolation was introduced in Python v.3.6 based on PEP-498. It's not supported by earlier versions of Python, so it would need to be restated using the somewhat less-elegant/less-readable backwards-compatible syntax of:

``k = 'attributes_fc%stc%sfj%stj%sfy%sty%sfd%std%sep%s' % (str(from_csv), str(to_csv), str(from_json), str(to_json), str(from_yaml), str(to_yaml), str(from_dict), str(to_dict), str(exclude_private))``

or something similar.

Reply:
done